core/include/Spirit/Exception.h

```diff
@@ -5,4 +5,5 @@
 #define Exception_Simulated_domain_too_small 3
 #define Exception_Not_Implemented            4
 #define Exception_Non_existing_Image         5
-#define Exception_Non_existing_Chain         6
\ No newline at end of file
+#define Exception_Non_existing_Chain         6
+#define Exception_File_reading_error         7
```

Comment:
This sounds a little too similar to `Exception_File_not_Found`. Maybe `Exception_Bad_File_Content` would be better?

Reply:
You are right. Will be renamed with next commit.